src/quo/foundations/shadows.cljs

```diff
@@ -5,60 +5,59 @@
     [quo.theme :as quo.theme]
     [react-native.platform :as platform]))
 
+(defn clamp
```

Comment:
Similar to `utils.number/value-in-range`, this function can/should live in `utils.number`. Consider adding a docstring as we do with some utility functions and unit test a few inputs.